helmfile/values/prometheus-alerts-sc.yaml.gotmpl

```diff
@@ -6,6 +6,12 @@ defaultRules:
   rules:
     alertmanager: true
     rookMonitor: {{ .Values.monitoring.rook.enabled }}
+    {{- if and .Values.thanos.enabled .Values.thanos.metrics.enabled }}
     thanosMonitor: {{ .Values.thanos.metrics.enabled }}
+    {{- end }}
+    {{- if not .Values.prometheus.predictLinearLimit }}
+    predictLinear: false
+    {{- end }}
```

Comment:
Will this be set to false if you set `predictLinearLimit: 0` or is it `predictLinearLimit: null`? Either way, could you add a comment in wc-config and sc-config to explain this option there?

Reply:
I've updated the comment about that option.